src/test/java/org/opentosca/container/client/ContainerClientTests.java

```diff
@@ -1,53 +1,46 @@
 package org.opentosca.container.client;
 
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
 import io.swagger.client.model.ServiceTemplateInstanceDTO;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.eclipse.winery.accountability.exceptions.AccountabilityException;
+import org.eclipse.winery.repository.exceptions.RepositoryCorruptException;
 import org.junit.Assert;
-import org.junit.Assume;
-import org.junit.Before;
 import org.junit.FixMethodOrder;
 import org.junit.Test;
-import org.junit.runner.RunWith;
 import org.junit.runners.MethodSorters;
-import org.opentosca.container.client.model.Application;
-import org.opentosca.container.client.model.ApplicationInstance;
-import org.opentosca.container.client.model.BoundaryDefinitionProperties;
-import org.opentosca.container.client.model.Plan;
-import org.opentosca.container.client.model.PlanInstance;
-import org.opentosca.container.client.model.PlanType;
-import org.opentosca.container.client.model.PropertyMapping;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.opentosca.container.client.model.*;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringRunner;
 
-@RunWith(SpringRunner.class)
+import javax.xml.namespace.QName;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
 @SpringBootTest
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 public class ContainerClientTests {
 
-    @Autowired
-    private ClientTests.Config config;
-
+    public static final String TESTAPPLICATIONSREPOSITORY = "https://github.com/OpenTOSCA/tosca-definitions-test-applications";
```

Comment:
I would set all possible fields to `final`

Reply:
fixed